### DISCUSSION TITLE
How to omit folders from "-use-full-path-album-name"

### DISCUSSION THREAD

**Discussion by thimplicity** [simulot/immich-go#563] [Q&A]
Hi, I would like to transition to Immich from OneDrive. I have a three folder structure today: topic/year/location. My understanding is that with "-create-album-folder", it would create albums with the "location" folder. With "-use-full-path-album-name" and a separator "-" it would do "topic-year-location". Two questions:
1) Is there a way to only use the "topic" folder?
2) Is there a way to have it add the picture to three albums with the three folder?

Thanks!

**Comment by simulot** [comment]
There isn't direct way to add photos `topic/year/location`  automatically to an album named after the the first part of the path.

You can add all photos from `topic/year/location` to the album `topic` with the option `-album="topic"`
You can script it to automatize the process for all topics.

The import is recursive by default. All photos from the `topic` folder and its sub-folders will be uploaded
